views/editQuiz.handlebars

```diff
@@ -0,0 +1,53 @@
+<head>
+    <meta charset="utf-8"/>
+    <title>Quizicle</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" crossorigin="anonymous">
+    <link rel="stylesheet" href="/style.css">
+</head>
+
+<body>
+    {{> header}}
+
+    <div class="edit_quiz">
+        <div class="intro">
+	    <div class="edit_title">
+	        <input id="edit-quiz-title-input" placeholder="Title..."></input>
+	    </div>
+	    <div class="edit_description">
+	        <textarea id="edit-quiz-description-input" placeholder="Description..."></textarea>
+	    </div>
+	    <div class="edit-quiz-tags">
+	        <div class="edit-tags-par">
+	            <p>Tags (separated by space)</p>
+		</div>
+		<div class="edit-quiz-tags-input">
+		    <input id="edit-tags-input" placeholder="Tags..."></input>
+		</div>
+	    </div> 
+        </div>
+	<div class="edit-study_cards">
+	    <div class="edit-questions">
```

Comment:
Please refractor your use of divs to have as few divs as possible.

Reply:
I tried to refactor this as little bit and I think something like this might be a better structure:
```
<body>

    {{> header}}

    <div class="content_container edit_quiz_header">
        <input class="edit_quiz_title_input" placeholder="Title..."></input>
        <textarea class="edit_quiz_description_area" placeholder="Description..."></textarea>
        <p class="edit_quiz_tags_label">Tags (separated by space)</p>
        <input class="edit_quiz_tags_input" placeholder="Tags..."></input>
    </div>
    <div class="content_container header">
        <p>Edit Questions</p>
    </div>
    <div class="content_container edit_questions_container">
        <div class="edit_card">
            <button type="button" class="clickable remove_card_button">x</button>
            <textarea class="edit_card_prompt_area" placeholder="Prompt..."></textarea>
            <textarea class="edit_card_prompt_area" placeholder="Answer..."></textarea>
        </div>
        <div class="append_card">
            <button type="button" class="clickable add_card_button">+ Add Card</button>
        </div>
        <div class="edit_quiz_footer">
            <button type="button" class="clickable cancel_button">Cancel</button>
            <button type="button" class="clickable done_button">Done</button>
        </div>
    </div>

</body>
```